golang/redhat/crud/booster.yaml

```diff
@@ -0,0 +1,24 @@
+metadata:
+  app:
+    launcher:
+      runsOn:
+        - '!starter'
+name: Red Hat GoLang - CRUD Example
+description: Runs a Golang application exposing a HTTP endpoint proposing CRUD operations
+source:
+  git:
+    url: https://github.com/golang-starters/golang-http-crud
+    ref: master
+metadata:
+  app:
+    osio:
+      enabled: true
+environment:
+  staging:
+    source:
+      git:
+        ref:
```

Comment:
You need values for these entries. If there is none, you can safely remove the entire `environment:` entry

Reply:
Thanks George. Does this apply to the production entries as well?